lib/Timber.php

```diff
@@ -428,28 +428,6 @@ public static function get_menu( $menu = null, array $options = [] ) {
 		return $factory->from($menu, $options);
 	}
 
-	public static function _get_menu_id_from_locations( $slug, $locations ) {
-		if ( isset($locations[$slug]) ) {
-			$menu_id = $locations[$slug];
-			// TODO do this in filter/Integration
```

Comment:
I see that you removed the special WPML check and just want to confirm: Can we really just ditch this? It was added in #1437.

The tests that we have would suggest that this still works. But if we still need to check this thoroughly, we can maybe do it in a separate issue.

Reply:
Ack, you're right, I missed closing that loop because all the tests still passed.

@jarednova what needs to happen here? I'm not familiar with WPML and I'm not sure why it wouldn't fail once we take out those lines.

Reply:
@acobster I was scratching my head over that this morning too. Unless it breaks something in your work (or requires additional refactor), I'd like to restore these lines here ...

```php
if ( function_exists('wpml_object_id_filter') ) {
				$menu_id = wpml_object_id_filter($locations[$slug], 'nav_menu');
			}
```

We can then take it up on a separate ticket or PR about whether those tests are still valid, or perhaps some other check is necessary. This was a fix contributed specifically by the WPML team, so we can also follow-up with them if something in their code might have changed to make this no longer necessary.

Reply:
Restored as a proper integration. I had to change the mock `wpml_object_id_filter()` implementation in `test-timber-wpml.php` but I think that's only because `TestMenuFactory::testGetMenuFromLocation()` introduced some coverage that wasn't there before: getting a `Menu` by _location name_ when the hard-coded `extra-menu` location is not defined:

```php
function wpml_object_id_filter( $element_id, $element_type = 'post', $return_original_if_missing = false, $language_code = null ) {
	$locations = get_nav_menu_locations();
	if (isset($locations['extra-menu'])) {
		return $locations['extra-menu'];
	}
	return $element_id; // <--- had to add this
}
```